plugins/permission-common/src/permissions/util.ts

```diff
@@ -75,3 +86,26 @@ export function isUpdatePermission(permission: Permission) {
 export function isDeletePermission(permission: Permission) {
   return permission.attributes.action === 'delete';
 }
+
+export function toPermissionEvaluator(
+  permissionAuthorizer: PermissionAuthorizer,
+): PermissionEvaluator {
+  return {
+    authorize: async (
+      requests: AuthorizePermissionRequest[],
+      options?: EvaluatorRequestOptions,
+    ): Promise<AuthorizePermissionResponse[]> => {
+      const response = await permissionAuthorizer.authorize(requests, options);
+
+      return response as DefinitivePolicyDecision[];
+    },
+    query(
+      requests: QueryPermissionRequest[],
+      options?: EvaluatorRequestOptions,
+    ): Promise<QueryPermissionResponse[]> {
+      // @ts-expect-error
```

Comment:
Perhaps cast instead here? Just to make sure no other errors are being ignored and such. This kinda also hides the intention, as in it's not always obvious which error we're looking to ignore